### ISSUE TITLE
Missing dependency

### ISSUE THREAD

**Issue by LCC-gblack** [raspberry-vanilla/android_kernel_manifest#17]
TL;DR. rsync is a dependency for this build.

I was putting this repo in docker and getting this error upon building:
ERROR: An error occurred during the fetch of repository '_main~kernel_toolchain_ext~kleaf_host_tools':
   Traceback (most recent call last):
        File "/kernel-dev/build/kernel/kleaf/impl/declare_host_tools.bzl", line 50, column 31, in _kleaf_host_tools_repo_impl
                repository_ctx.symlink(repository_ctx.which(host_tool), host_tool)
Error in symlink: in call to symlink(), parameter 'target' got value of type 'NoneType', want 'string, Label, or path'

Looking in declare_host_tools.bzl, there is:
_DEFAULT_HOST_TOOLS = [
    "bash",
    "perl",
    "rsync",
    "sh",
    # For BTRFS (b/292212788)
    "find",
]
My docker was missing rsync.  I confirmed it is not on the list of things to install in the AOSP development environment link.

The build worked after I installed rsync.

**Comment by KonstaT** [comment]
Thanks for the report but this is likely not the right place for this.

Google's documentation on building in kernels is somewhat lacking (https://source.android.com/docs/setup/build/building-kernels). Actually, it doesn't mention setting up the build environment at all. On Raspberry Vanilla project's documentation on building AOSP (https://github.com/raspberry-vanilla/android_local_manifest), I've included an additional step (2. Install additional packages) that is required due to changes/additions this project makes to plain AOSP. In case of the kernel, dependency on `rsync` is not due to any changes made withing this project.

Correct place to report this would likely be Google's issue tracker.
https://source.android.com/docs/setup/contribute/report-bugs#documentation

**Comment by LCC-gblack** [comment]
Fair enough.  I thought you might want to update the readme instructions is all.  I'll look into reporting this in the correct location.
